tests/test-sources/plugins/lazyloading/lz-n.nix

```diff
@@ -0,0 +1,337 @@
+let
+  getFirstLznPlugin =
+    config:
+    let
+      inherit (config.plugins.lz-n) plugins;
+    in
+    if plugins == [ ] then null else builtins.head plugins;
+
+  getPluginKeys = plugin: if plugin != null && builtins.isList plugin.keys then plugin.keys else [ ];
+in
+{
+  lazy-load-neovim-plugin-configured =
+    { config, lib, ... }:
+    {
+      plugins = {
+        lz-n = {
+          enable = true;
+        };
+
+        neotest = {
+          enable = true;
+          lazyLoad = {
+            enable = true;
+            settings = {
+              keys = [
+                {
+                  __unkeyed-1 = "<leader>nt";
+                  __unkeyed-3 = "<CMD>Neotest summary<CR>";
+                  desc = "Summary toggle";
+                }
+              ];
+            };
+          };
+        };
+      };
+
+      assertions =
```

Comment:
Q: is it intended that (all?) these assertion-based tests also run `nvim` as part of the test? Or should we set `test.runNvim = false` to make the tests cheaper?

Reply:
Good callout, we don't really need to run nvim for these test scenarios since we're just trying to make sure we pass things around properly.